test/library/standard/CSV/psahabu/fremont_bike_counts.csv

```diff
@@ -0,0 +1,1001 @@
+Date,Fremont Bridge West Sidewalk,Fremont Bridge East Sidewalk
```

Comment:
I don't know the licensing terms for this data.  Do we have permission to redistribute it?

Reply:
It's public domain. https://data.seattle.gov/Transportation/Fremont-Bridge-Hourly-Bicycle-Counts-by-Month-Octo/65db-xm6k